pkg/pillar/kubeapi/storageutils.go

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build kubevirt
+
+package kubeapi
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/lf-edge/eve/pkg/pillar/pubsub"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// XXX current used module version has int32, latest has uint64
+func device_major(stat syscall.Stat_t) int32 {
+	return int32((stat.Rdev >> 8) & 0xfff)
+}
+func device_minor(stat syscall.Stat_t) int32 {
+	return int32((stat.Rdev & 0xff) | ((stat.Rdev >> 12) & 0xfff00))
+}
+func getMajorMinor(stat syscall.Stat_t) string {
+	major := device_major(stat)
+	minor := device_minor(stat)
+	return fmt.Sprintf("%d:%d", major, minor)
+}
+
+func Longhorn_PVCs_Exist() bool {
+	LonghornDevPath := "/dev/longhorn"
```

Comment:
Maybe define this in the front of the file, to see if the variable not using 'longhorn'?

Reply:
moved to a file const